### PR TITLE
Fix misc. WOCCatalog breaks in Xcode

### PR DESCRIPTION
**Pull request by s5msft**
Various (small) fixes to parts of WOCCatalog when built for iOS

### CODE REVIEW

**Comment by Ramu-msft** [comment]
@bdrlamb is added to the review. #Closed

**Comment by Ramu-msft** [comment]
@oliversa-msft is added to the review. #Closed

**Comment by Ramu-msft** [comment]
@davelamb is added to the review. #Closed

**Comment by oliversa-msft** [comment]
:clock1:

**Comment by bdrlamb** [comment]
:shipit:

File: samples/WOCCatalog/WOCCatalog/WebViewController.m [hunk 1]
```diff
@@ -34,7 +34,7 @@ - (void) viewDidLoad {
     self.webView.delegate = self;
     [self.view addSubview:self.webView];
     
-    [self.webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"http://www.microsoft.com/"]]];
+    [self.webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://www.microsoft.com/"]]];
```
Comment (davelamb) [inline samples/WOCCatalog/WOCCatalog/WebViewController.m]: Phew, glad we fixed this before WOCCatalog got security audited. :)

**Comment by davelamb** [comment]
:shipit:

**Comment by Ramu-msft** [comment]
Thanks for your contribution.

**Pull request merged**